src/System.Data.Common/tests/System/Xml/XmlDataDocumentTests.cs

```diff
@@ -0,0 +1,154 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data;
+using System.Data.Tests;
+using System.IO;
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class XmlDataDocumentTests
+    {
+        [Fact]
+        public static void XmlDataDocument_Ctor()
+        {
+            XmlDataDocument doc;
+            doc = new XmlDataDocument();
+            doc = new XmlDataDocument(new DataSet());
```

Comment:
Is there anything that can be validated here?  For example, what should doc.DataSet be with the default ctor?  What should it be when passing in a DataSet?